bowling-ball/Game.cs

```diff
@@ -1,19 +1,62 @@
-﻿using System;
+using System;
 
 namespace BowlingBall
 {
     public class Game
     {
-        public void Roll(int pins)
-        {
-            throw new NotImplementedException();
-        }
+        Frames [] frames = new Frames[10];
+        int PointerToFrame;
+        const int TotalPossibleFrames = 10;
+        int ActualFrames;
+        public Game()
+        {
+            PointerToFrame = 0;
 
-        public int GetScore()
+            for (int index = 0; index < TotalPossibleFrames; index++)
+                frames[index] = new Frames();
+        }
+        public void Roll(int [] pins)
         {
-            throw new NotImplementedException();
+            const int StrikeConstant = 10;
+            int ptr = 0;
+           while(ptr < pins.Length)
+            {
+                // to check if user is still playing currentframes
+                if (ptr + 1 == pins.Length)

```

Comment:
not a readable code. did not got what you are trying to do here. Please elaborate in detail.